rclcpp_components/src/component_manager.cpp

```diff
@@ -96,7 +97,7 @@ ComponentManager::get_component_resources(
     }
 
     std::string library_path = parts[1];
-    if (!rcpputils::fs::path(library_path).is_absolute()) {
+    if (!std::filesystem::path(library_path).is_absolute()) {
       library_path = base_path + "/" + library_path;
     }
     resources.push_back({parts[0], library_path});
```

Comment:
You could use `library_path` as a `std::filesystem::path` and then save it as a `std::string` at the end:

```
    std::filesystem::path library_path = parts[1];
    if (!library_path.is_absolute()) {
      library_path = (base_path / library_path);
    }
    resources.push_back({parts[0], library_path.string()});
```